js/index.js

```diff
@@ -67,3 +67,38 @@ navLinks.forEach(link => {
     });
 });
 
+// DOM Manipulation for Footer
+
+const footer = document.createElement('footer');
+document.body.appendChild(footer);
+
+const today = new Date();
+const thisYear = today.getFullYear();
+
+const copyright = document.createElement('p');
```

Comment:
The stretch goal of including the copyright symbol using Unicode.

`const copyrightSymbol = "\u00A9";`
`const copyrightContent = `${copyrightSymbol} Daria Bolgova ${thisYear}`;